### DISCUSSION TITLE
selenium-stealth integration with SB UC mode

### DISCUSSION THREAD

**Discussion by cluebeatz** [seleniumbase/SeleniumBase#2856] [Q&A] [answered]
hello I noticed in the repo issues of ways to integrate selenium stealth with seleniumbase but I didn't find examples of the integration with seleniumbase's SB mode, I managed to start the script by putting sb.driver but opening pixelscan I notice that the webgl values remain my own and the seleniumstealth values are not used,  i'm using chrome since brave is not supported by seleniumstealth.

`
 

          with SB(uc=True, incognito=True) as sb:
                stealth(
                    sb.driver,
                    languages=["en-US", "en"],
                    vendor="Google Inc.",
                    platform="Win32",
                    webgl_vendor="Intel Inc.",
                    renderer="Intel Iris OpenGL Engine",
                    fix_hairline=True,
                )

                url2test = "https://pixelscan.net/"
                sb.driver.uc_open_with_reconnect(url2test, 3)
                sb.sleep(120)
`

**Comment by mdmintz** [comment, marked as answer]
When UC Mode disconnects the driver from Chrome to evade detection, it also resets any settings made by `selenium-stealth`.

Also, it appears that using `selenium-stealth` to modify settings will get detected as a possible attempt to mask your fingerprint. (There is little that I can do there, as that is an external library.)

Your best bet to avoid detection is by just using regular SeleniumBase UC Mode:

```python
from seleniumbase import SB

with SB(uc=True, incognito=True) as sb:
    sb.uc_open_with_reconnect("https://pixelscan.net/", 10)
    breakpoint()
```

If you still want to use `selenium-stealth` with SeleniumBase, there's documentation here: https://github.com/seleniumbase/SeleniumBase/issues/2404
